src/components/common/send-notification/send-notification-service.ts

```diff
@@ -30,6 +31,10 @@ export function sendNotificationService(
       payload.phoneNumber = phoneNumber;
     }
 
+    if (requestNewCode) {
```

Comment:
Not a huge quibble, but I guess this could equally well be `payload.requestNewCode = requestNewCode || false` and avoid the `if` statement altogether, or `requestNewCode` could have a default value of false. It makes little practical difference, but in some way might be nicer always to send this field to the back-end? Don't feel strongly, and there may be reasons not to do this, but that'd be my line of thinking